server/services/zigbee2mqtt/lib/getDiscoveredDevices.js

```diff
@@ -3,21 +3,27 @@ const { convertDevice } = require('../utils/convertDevice');
 
 /**
  * @description Get discovered devices.
+ * @param {object} filters - Filters to apply.
  * @returns {Array} Array of discovered devices.
  * @example
  * getDiscoveredDevices();
  */
-function getDiscoveredDevices() {
-  return (
-    Object.values(this.discoveredDevices)
-      // Convert to Gladys device
-      .map((d) => convertDevice(d, this.serviceId))
-      .map((d) => {
-        const existingDevice = this.gladys.stateManager.get('deviceByExternalId', d.external_id);
-        // Merge with existing device.
-        return mergeDevices(d, existingDevice);
-      })
-  );
+function getDiscoveredDevices(filters = {}) {
+  let devices = Object.values(this.discoveredDevices)
+    // Convert to Gladys device
+    .map((d) => convertDevice(d, this.serviceId))
+    .map((d) => {
+      const existingDevice = this.gladys.stateManager.get('deviceByExternalId', d.external_id);
+      // Merge with existing device.
+      return mergeDevices(d, existingDevice);
+    });
+
+  const { filterExisting } = filters;
```

Comment:
Can you put this in snake_case (only in the REST API)

The API is snake_case only!

Reply:
Ok, done, see the last commit ;)
I'll have to change LANManager to do same change :)